src/client/pythonEnvironments/base/info/env.ts

```diff
@@ -180,6 +180,12 @@ function buildEnvDisplayString(env: PythonEnvInfo, getAllDetails = false): strin
     if (env.name && env.name !== '') {
         envSuffixParts.push(`'${env.name}'`);
     }
+    if (env.location && env.location !== '') {
```

Comment:
```suggestion
    else if (env.location && env.location !== '') {
```
We don't want to do this for "named" conda environments, i.e only do this when name is not set.

Reply:
Also, there are some failing test, it's probably related to this.